### DISCUSSION TITLE
when upload several large files, any way to small footprint?

### DISCUSSION THREAD

**Discussion by yyq2013aa** [go-resty/resty#675] [General]
`resp, err := client.R(). SetFiles(map[string]string{ "profile_img": "/Users/jeeva/test-img.png", "notes": "/Users/jeeva/text-file.txt", }). Post("http://myapp.com/upload") `

It seems read all files bytes in memory, any functions to avoid it?

**Comment by jeevatkm** [comment]
Aim for this feature in the v3.0.0. Thanks.
